cedar-policy-core/src/est.rs

```diff
@@ -356,7 +356,7 @@ mod test {
         let ast = est
             .try_into_ast_policy(None)
             .expect("Failed to convert to AST");
-        ast.try_into().expect("Failed to convert to EST")
```

Comment:
Not sure if we actually want this change.

Reply:
I think this is fine.  Probably the AST->EST transform was originally fallible but now is not (as we can see from there now being an `Into` impl)?

Reply:
Seems legit. We have `impl From<ast::Policy> for Policy`, so the `TryFrom` should be automatic from that